examples/xdp_drop_count.rs

```diff
@@ -0,0 +1,110 @@
+extern crate bcc;
```

Comment:
This isn't needed with 2018 edition syntax.

Reply:
Removed!